hw/ip/pattgen/data/pattgen.hjson

```diff
@@ -0,0 +1,157 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+{ name: "pattgen",
+  clock_primary: "clk_i",
+  bus_device: "tlul",
+  bus_host: "none",
+  regwidth: "32",
+  // OUTPUT pins
+  available_output_list: [
+    { name: "pda0_tx", desc: "Serial output data bit for pattern generation on Channel 0" }
+    { name: "pcl0_tx", desc: "Clock corresponding to pattern data on Channel 0" }
+    { name: "pda1_tx", desc: "Serial output data bit for pattern generation on Channel 1" }
+    { name: "pcl1_tx", desc: "Clock corresponding to pattern data on Channel 1" }
+  ]
+  // INTERRUPT pins
+  interrupt_list: [
+    { name: "done_ch0"
+      desc: "raise if pattern generation on Channel 0 is complete"
+    }
+    { name: "done_ch1"
+      desc: "raise if pattern generation on Channel 1 is complete"
+    }
+  ],
+  param_list: [
+    { name:    "NumRegsData",
+      type:    "int",
+      default: "2",
+      desc:    "Number of data registers per each channel",
+      local:   "true"
+    }
+  ],
+  // REGISTER definition
+  registers: [
+    // CTRL register
+    { name: "CTRL",
+      desc: "PATTGEN control register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      fields: [
+        { bits: "0",
+          resval: "0",
+          name: "ENABLE_CH0",
+          desc: "Enable pattern generator functionality for Channel 0"
+        }
+        { bits: "1",
+          resval: "0",
+          name: "ENABLE_CH1",
+          desc: "Enable pattern generator functionality for Channel 1"
+        }
+        { bits: "2",
+          resval: "0",
+          name: "POLARITY_CH0",
+          desc: "Clock (`pcl`) polarity for Channel 0.  If high, `pda` signal changes on falling edge of pcl signal, otherwise pda changes on rising edge. Note that writes to a channel's configuration registers have no effect while the channel is enabled."
+        }
+        { bits: "3",
+          resval: "0",
+          name: "POLARITY_CH1",
+          desc: "Clock (`pcl`) polarity for Channel 1.  If high, `pda` signal changes on falling edge of `pcl` signal, otherwise pda changes on rising edge. Note that writes to a channel's configuration registers have no effect while the channel is enabled."
+        }
+      ]
+    }
+    { name: "PREDIV_CH0",
+      desc: "PATTGEN pre-divider register for Channel 0",
+      count: "NumRegsPrediv",
+      cname: "PATTGEN",
```

Comment:
Lints 66-67 and 81-82 should be removed since PREDIV_CH0 and PREDIV_CH1 is a single register

Reply:
I see what you are saying.  Will be removed in next commit.